wagtail/embeds/tests/test_embeds.py

```diff
@@ -572,3 +574,40 @@ def test_clean_invalid_url(self, get_embed):
         with self.assertRaises(ValidationError):
             required_block.clean(
                 EmbedValue('http://no-oembed-here.com/something'))
+
+
+class EmbedForm(forms.ModelForm):
+    class Meta:
+        model = Embed
+        fields = ["url", "max_width"]
+
+
+class TestEmbedModel(TestCase):
+    def test_url_fields(self):
+        self.assertIsInstance(Embed._meta.get_field('url'), URLField)
+        self.assertIsInstance(Embed._meta.get_field("thumbnail_url"), URLField)
```

Comment:
This test can be safely removed, I think. It would be very odd indeed for Django to not respect our model field definitions.